webapi/src/main/resources/application.conf

```diff
@@ -9,6 +9,7 @@ pekko {
     logging-filter = "org.apache.pekko.event.slf4j.Slf4jLoggingFilter"
     log-dead-letters = off
     log-dead-letters-during-shutdown = off
+    logger-startup-timeout = 30s
```

Comment:
Let's first try if only this configuration change solves your problem.

From the logs I can see that the ` Slf4jLogger started` takes roughly seven seconds, so a timeout of 15 seconds should be more than sufficient.

If the ActorSystem does not startup properly I do not want it to be retried, instead it should fail immediately. Please remove the retry from its `acquire`

Reply:
Let's try it out.